tests/server_test.go

```diff
@@ -109,7 +109,7 @@ func withIdleBulkTime(idleBulkTime time.Duration) serverOption {
 }
 
 func withDelimiter(delimiter string) serverOption {
```

Comment:
I am not sure what was the original motivation for `string -> rune` but I think we should keep it consistent

```suggestion
func withDelimiter(delimiter rune) serverOption {
```